packages/sanes-chrome-extension/src/routes/requests/components/RequestList.tsx

```diff
@@ -43,12 +60,19 @@ const RequestList = ({ requests }: Props): JSX.Element => {
         {requests.map((req: Request, index: number) => {
           const first = index === 0;
           const firstElemClass = first ? classes.first : undefined;
+          const onRequestClick = buildClickHandlerFrom(req);
+          const text = req.type === 'getIdentities' ? 'Get Identities Request' : 'Sign Request';
 
           return (
             <React.Fragment>
-              <ListItem key={`${index}`} className={firstElemClass} disabled={!first}>
+              <ListItem
+                key={`${index}`}
+                className={firstElemClass}
+                disabled={!first}
+                onClick={first ? onRequestClick : undefined}
+              >
                 <ListItemText
-                  primary="Type of request"
+                  primary={text}
```

Comment:
I think we should show who is asking for the permissions (resource name). Because there is no information about source of the request. I understand that customer will see this info on next screen, but I think it is better for reviewing purpose.

Reply:
Sure, I will keep in mind, we can discuss it when #201 is done and surrounding tests we need to implement.